datasets/lj_speech/README.md

```diff
@@ -187,7 +187,7 @@ Recordings by Linda Johnson from LibriVox. Alignment and annotation by Keith Ito
 
 ### Personal and Sensitive Information
 
-[Needs More Information]
+The dataset consists of people who have donated their voice online. You agree to not attempt to determine the identity of speakers in this dataset.
```

Comment:
Can be rephrased/omitted for LJ Speech IMO, since these recordings are done by the authors themselves, and they released them into the public domain.